tests/unit/dom/select-test.js

```diff
@@ -0,0 +1,209 @@
+import { module, test } from 'qunit';
+import { select, setupContext, teardownContext } from '@ember/test-helpers';
+import { buildInstrumentedElement } from '../../helpers/events';
+import hasEmberVersion from '@ember/test-helpers/has-ember-version';
+import { isIE11 } from '../../helpers/browser-detect';
+
+let clickSteps = ['focus', 'focusin', 'input', 'change'];
```

Comment:
Can you rename this variable? 
```suggestion
let selectSteps = ['focus', 'focusin', 'input', 'change'];
```